src/TailoringWindow.cpp

```diff
@@ -131,11 +131,11 @@ TailoringWindow::TailoringWindow(struct xccdf_policy* policy, struct xccdf_bench
 
     QObject::connect(
         mUI.itemsTree, SIGNAL(currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)),
-        this, SLOT(itemSelectionChanged(QTreeWidgetItem*,QTreeWidgetItem*))
+        this, SLOT(itemSelectionChanged(QTreeWidgetItem*))
     );
     QObject::connect(
-        mUI.itemsTree, SIGNAL(itemChanged(QTreeWidgetItem*,int)),
-        this, SLOT(itemChanged(QTreeWidgetItem*,int))
+        mUI.itemsTree, SIGNAL(itemChanged(QTreeWidgetItem*, int)),
```

Comment:
Sorry about the spare whitespace change here. Could someone more familiar with the QT framework explain why it was necessary that this have the "int" parameter while it is marked unused later (on line 138)?

Reply:
I think we may need to keep the int. It's necessary for the slot signal connection to happen. moc needs it to choose the correct method. In essence you need to use full method names and since C++ supports overloading you need to include parameters.